frontend/src/hooks/observable.tsx

```diff
@@ -7,7 +7,7 @@ import { Observable } from "rxjs";
  *
  * @param {Observable} observable - Observable used to change state
  */
-function useObservable(observable: Observable<boolean>) {
+function useObservable(observable: Observable<boolean | string | null>) {
```

Comment:
Do not hardcode type here. Please use template type or any type:
`function useObservable<T>(observable: Observable<T>) {
}`

Reply:
Fixed.
Why hardcoding is a bad idea?